src/pybind/ceph_volume_client.py

```diff
@@ -904,7 +905,7 @@ def _volume_metadata_set(self, volume_path, data):
         that encoded the metadata.
         """
         data['compat_version'] = 1
-        data['version'] = 1
+        data['version'] = 2
```

Comment:
Somewhat unrelated to this change, but maybe this line should be using self.version instead of repeating the literal `2`

Reply:
Done